draft-ietf-quic-recovery.md

```diff
@@ -1142,6 +1142,21 @@ Pseudocode for OnPacketSent follows:
      SetLossDetectionTimer()
 ~~~
 
+## On Receiving a Packet
+
+When a packet is received, it may allow the server to send
+if the server was previously amplification limited.
+
+Psuedocode for OnPacketReceived follows:
+
+~~~
+OnPacketReceived(packet):
+  // If this packet unblocks the server, arm the
+  // timer to ensure forward progress resumes
+  if (server was at anti-amplification limit):
+    SetLossDetectionTimer()
+  DecryptAndProcessFrames()
```

Comment:
This doesn’t seem to belong in recovery. I think you’re trying to point out that even undecryptable packets unblock the server, right?

Reply:
Yeah, I think this should be in the comments above.